config.yaml

```diff
@@ -345,7 +346,6 @@ languages:
     versions:
       current: v1.0.0
       list:
-      - v1.0.0
```

Comment:
Why did you remove this line?

Reply:
Sorry, I accidentally missed that part while updating the `config.yaml` and fixed it.

Thanks